src/models/User.ts

```diff
@@ -266,23 +284,35 @@ const userSchema = new Schema(
     },
     tokenVersion: {
       type: Number,
-      required: true,
       default: 0,
```

Comment:
Why have you removed required from this ?

Reply:
fixed it sir